mobile/library/swift/EnvoyCxxSwiftInterop/cxx_swift_interop.h

```diff
@@ -0,0 +1,43 @@
+#import "library/cc/bridge_utility.h"
+#import "library/cc/direct_response_testing.h"
+#import "library/cc/engine_builder.h"
+#import "library/common/data/utility.h"
+#import "library/common/extensions/filters/http/platform_bridge/c_types.h"
+#import "library/common/main_interface.h"
+#import "library/common/network/apple_platform_cert_verifier.h"
+#import "library/common/stats/utility.h"
+
+namespace Envoy {
+namespace CxxSwift {
+
+using StringVector = std::vector<std::string>;
+using StringPair = std::pair<std::string, std::string>;
+using StringPairVector = std::vector<StringPair>;
+using StringMap = absl::flat_hash_map<std::string, std::string>;
+using HeaderMatcherVector = std::vector<DirectResponseTesting::HeaderMatcher>;
+using BootstrapPtr = intptr_t;
+
+inline void string_map_set(StringMap& map, std::string key, std::string value) { map[key] = value; }
+
+inline void raw_header_map_set(Platform::RawHeaderMap& map, std::string key,
+                               std::vector<std::string> value) {
+  map[key] = value;
+}
+
+inline const DirectResponseTesting::MatchMode DirectResponseMatchModeContains =
+    DirectResponseTesting::contains;
+inline const DirectResponseTesting::MatchMode DirectResponseMatchModeExact =
+    DirectResponseTesting::exact;
+inline const DirectResponseTesting::MatchMode DirectResponseMatchModePrefix =
+    DirectResponseTesting::prefix;
+inline const DirectResponseTesting::MatchMode DirectResponseMatchModeSuffix =
+    DirectResponseTesting::suffix;
+
+inline BootstrapPtr generateBootstrapPtr(Platform::EngineBuilder builder) {
+  return reinterpret_cast<BootstrapPtr>(builder.generateBootstrap().release());
```

Comment:
Interesting. builder.generateBootstrap() returns a unique_ptr which we release into a raw pointer which we then cast to an int. Is that right? I think it would be good to add a comment which explains this.